template/roster.hbs

```diff
@@ -5,29 +5,33 @@
          data-presence-offline-label="{{t "You_are_currently_offline"}}"></ul>
    </div>
    <div class="jsxc-roster-status"></div>
-   <div class="jsxc-bottom jsxc-presence jsxc-roster-item jsxc-bar" data-bid="own">
-      <div class="jsxc-avatar jsxc-shrink--no"></div>
-      {{#> menu
-        classes="jsxc-menu--pushup jsxc-grow jsxc-js-presence-menu"
-        button-label=(t "Offline")}}
-            <li data-presence="online" class="jsxc-online jsxc-offline-available">{{t "Online"}}</li>
-            <li data-presence="chat" class="jsxc-chat">{{t "Chatty"}}</li>
-            <li data-presence="away" class="jsxc-away">{{t "Away"}}</li>
-            <li data-presence="xa" class="jsxc-xa">{{t "Extended_away"}}</li>
-            <li data-presence="dnd" class="jsxc-dnd">{{t "dnd"}}</li>
-            <li data-presence="offline" class="jsxc-offline">{{t "Offline"}}</li>
-      {{/menu}}
+   <div class="jsxc-real-bottom">
```

Comment:
Real bottom was probably not intentionally included in this pr.

Reply:
right... C&P error ;D